src/main/resources/templates/db_scripts/updateItemStatusDate.sql

```diff
@@ -0,0 +1,23 @@
+
+-- Updates item status date if status name was changed.
+-- Item status date stores with timezone.
+CREATE OR REPLACE FUNCTION ${myuniversity}_${mymodule}.update_status_date() RETURNS TRIGGER
+AS $$
+  DECLARE
+	  newStatus text;
+  BEGIN
+  	newStatus = NEW.jsonb->'status'->>'name';
+  IF newStatus IS NULL THEN  RETURN NEW;
+  END IF;
+	IF (TG_OP='UPDATE') THEN
+	  IF (newStatus!=OLD.jsonb->'status'->>'name') THEN
```

Comment:
I'm not sure if this will work for case when either newStatus or oldStatus is null. Take a look at `IS DISTINCT FROM` operator.

Reply:
Thank you for advice. Done